library/watermarks.schema.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<list xml:id="mei_5607df02-a8c8-4760-a721-4336707cc0da" xmlns="http://www.music-encoding.org/ns/mei" xmlns:xl="http://www.w3.org/1999/xlink">
+	<head>Persons</head>
```

Comment:
I suppose the <head> should read „Watermarks“, right? ;-) 
@peterstadler what do you think – I wonder if this specific list is universally useful to have it in the branch of merMEId code? If yes, perfectly fine with me, otherwise I'd propose to shrink this to a dummy list showing the structure of the expected document and keep the actual list of watermarks with the project data. 
We might also keep it here for the moment but move it to a project-data repository in the future and inject it on installation time of the data xar package.